modules/nf-core/severus/meta.yml

```diff
@@ -0,0 +1,128 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "severus"
+description: Severus is a somatic structural variation (SV) caller for long reads (both PacBio and ONT)
+keywords:
+  - structural
+  - variation
+  - somatic
+  - germline
+  - long-read
+tools:
+  - "severus":
+      description: "A tool for somatic structural variant calling using long reads"
+      homepage: "https://github.com/KolmogorovLab/Severus"
+      documentation: "https://github.com/KolmogorovLab/Severus"
+      tool_dev_url: "https://github.com/KolmogorovLab/Severus"
+      doi: "10.1101/2024.03.22.24304756"
+      licence: ["BSD-3-clause"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - target_bam:
+      type: file
+      description: path to one or multiple target bam files (e.g. tumor, must be indexed)
+      pattern: "*.bam"
+  - target_index:
+      type: file
+      description: path to one or multiple target bam index files
+      pattern: "*.bai"
+  - control_bam:
+      type: file
+      description: path to the control bam file (e.g. normal, must be indexed)
+      pattern: "*.bam"
+  - control_index:
+      type: file
+      description: path to the control bam file index
+      pattern: "*.bai"
+  - vcf:
```

Comment:
Does the tool need a tbi index for compressed vcf?

Reply:
But does it use it if it is present?

Reply:
No, I can't see that it does. It has no arguments to process certain regions so I guess that's why it doesn't use an index.

Reply:
I don't think so, but I'm not sure. There's no argument to process a region so I guess the index might not be used.